Adyen/UI/Form/Items/IssuerList/FormIssuerItemView.swift

```diff
@@ -0,0 +1,33 @@
+//
+// Copyright (c) 2022 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import UIKit
+
+internal final class FormIssuerItemView: FormTextItemView<FormIssuerListItem> {
```

Comment:
I believe this class/file is not needed, correct me if I am wrong.

Reply:
Yes, I have removed them. Thanks for pointing it out.